### ISSUE TITLE
Electrum password

### ISSUE THREAD

**Issue by stormburn** [spesmilo/electrum#5932]
I have an electrum wallet on my laptop, when I try and send bitcoin it asks me for a password. I have never used/created a password for electrum (other than my seed.)

I tried to restore from seed, but the next button is greyed out when i enter it. Indicating the seed is wrong, yet I copied it verbatim and have triple and quadruple checked it. NOTE: It is 25 words long, usually they are 12? Not sure if this is an issue.

So basically I don't know the password OR the seed, but I still have the wallet open in electrum. Do I have any hope of recovering? Or am i doomed?

It has a significant amount of money, so any help would be immensely appreciated.

Here is some basic info about the wallet: https://imgur.com/a/8ruXWSb
